components/header/header.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import style from './header.module.scss';
+import KedroLogo from '../../public/assets/logos/kedro';
+import DiscordLogo from '../../public/assets/logos/discord';
+import GithubLogo from '../../public/assets/logos/github';
+import BurgerIcon from '../../public/assets/icons/burger';
+import CloseIcon from '../../public/assets/icons/close';
```

Comment:
I don't think we need this `assets` folder. We could just go `/public/icons/filename`.